filament/backend/include/backend/platforms/VulkanPlatform.h

```diff
@@ -71,6 +71,9 @@ class VulkanPlatform : public Platform, utils::PrivateImplementation<VulkanPlatf
         // where the gpu only has one graphics queue. Then the client needs to ensure that no
         // concurrent access can occur.
         uint32_t graphicsQueueIndex = 0xFFFFFFFF;
+                bool debugUtilsSupported = false;
+        bool debugMarkersSupported = false;
+        bool multiviewSupported = false;
```

Comment:
I don't really think this scales well, but to unblock your work, we can do this for now, but note that I might change this in the future.